app/models/survey.rb

```diff
@@ -29,4 +29,19 @@ def as_indexed_json(_options = {})
       started_at: started_at.strftime("%F"),
     }.as_json
   end
+
+
+  def self.history_for_page(page, start_date, end_date)
```

Comment:
We might consider renaming this, `history_for_page` doesn't really convey what the method is going to return (for example: `count_by_date`).

Reply:
Oh much better name